dune/stuff/common/vector.hh

```diff
@@ -144,9 +144,9 @@ struct VectorAbstraction< Dune::Stuff::Common::FieldVector< K, SIZE > >
 
   static const  size_t static_size = SIZE;
 
-  static inline VectorType create(const size_t sz)
+  static inline VectorType create(const size_t /*sz*/)
   {
-    return VectorType(sz);
+    return VectorType();
   }
```

Comment:
Could you comment on why you removed the `sz`? I forwarded it to the ctor of our `Stuff::Common::FieldVector` on purpose, since there is a check, if `sz == SIZE`, which we would have to do here, otherwise.